src/tactic/ui/tools/ingest_wdg.py

```diff
@@ -1546,11 +1569,15 @@ def execute(my):
             if filenames == []:
                 raise TacticException('No sequences are found in files. Please follow the pattern of [filename] + [digits] + [file extension (optional)]. Examples: [abc_1001.png, abc_1002.png] [abc.1001.mp3, abc.1002.mp3] [abc_100_1001.png, abc_100_1002.png]')
 
+        if library_mode:
+            relative_dir = relative_dir + "/001"
```

Comment:
It's a numbering system for folders. We want to avoid directories hitting a maximum file limit, so for every FILE_LIMIT files, we create a new folder and increment the folder number. 

EX: 

/001 contains first FILE_LIMIT files in the ingest
/002 contains the next FILE_LIMIT files